backend/oracleobjectstorage/options.go

```diff
@@ -225,16 +221,18 @@ The minimum is 0 and the maximum is 5 GiB.`,
 		Advanced: true,
 	}, {
 		Name:     "memory_pool_flush_time",
-		Default:  memoryPoolFlushTime,
+		Default:  fs.Duration(time.Minute),
+		Hide:     fs.OptionHideBoth,
 		Advanced: true,
 		Help: `How often internal memory buffer pools will be flushed.
```

Comment:
In the s3 implementation I deleted the help text here except for the first line and added ` (no longer used)` on the end. Same with the use-mmap flag below